assets/app/view/game/part/location_name.rb

```diff
@@ -243,11 +243,12 @@ def l_up24
         end
 
         def l_center
-          {
-            region_weights: CENTER,
-            x: 0,
-            y: -delta_y / 2,
+          loc = { x: 0, y: -delta_y / 2 }
```

Comment:
this can just be all inline, why did you split it up?

Reply:
Didn't realize it could be all inline, will fix.